### DISCUSSION TITLE
Collapse versioning directives

### DISCUSSION THREAD

**Discussion by JacobCoffee** [lepture/shibuya#52] [Ideas]
In some libs like Flask or Click (?) when there are multiple versioning directives they are collapsed into a dropdown.

e.g., 

`.. versionadded: 1.0` would just be visible, but 
```
.. versionadded: 1.0
	New thing

.. versionchanged: 1.1
	Changed thing
	
.. deprecated: 2.0
	No more thing
```

Would be collapsed.

Currently we get just a bunch of admonitions:

<img width="1432" alt="image" src="https://github.com/lepture/shibuya/assets/45884264/d9195430-9653-4748-ab81-74635b252505">

it would be cool to autodetect multiple and throw them into sphinx-dropdown (or even prettier, `sphinx-togglebutton`)

**Comment by lepture** [comment]
It would be better to create a sphinx extension to group the version changes into a `<details>` DOM.

**Reply by JacobCoffee** [reply]
that makes sense!
